docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2795,6 +2795,19 @@ Result:
 
 Submits a signed transaction to the cluster for processing.
 
+Note that this doesn't guarantee the transaction is confirmed by the cluster.
+While this method reasonably retries to submit it background, the transaction
+could be rejected if transaction's `recent_blockhash` expires.
+
+In other words, this method immediately succeeds, without waiting for any
+confirmations.
+
+Use [`getSignatureStatuses`](jsonrpc-api.md#getsignaturestatuses) to make sure
+the transaction is confirmed or not.
+
+Also, this method doesn't alter the transaction in any way. This method just
+relays the transaction created by clients to the cluster as-is.
```

Comment:
```suggestion
This method does not alter the transaction in any way; it relays the transaction created by clients to the node as-is.

If the node's rpc service receives the transaction, this method immediately succeeds, without waiting for any confirmations. A successful response from this method does not guarantee the transaction is processed or confirmed by the cluster.

While the rpc service will reasonably retry to submit it, the transaction
could be rejected if transaction's `recent_blockhash` expires before it lands.

Use [`getSignatureStatuses`](jsonrpc-api.md#getsignaturestatuses) to ensure
a transaction is processed and confirmed.
```

Reply:
(Sorry for the lack of proper wrapping... github doesn't make that easy in suggestions :( )